### ISSUE TITLE
Installation issue.

### ISSUE THREAD

**Issue by alejandronanez** [ChrisTheBaron/cylon-rolling-spider#10]
Hi there, I'm trying to install this package and I'm getting some issues, here's my log:

```
 xpc-connection@0.1.4 install /Users/alejandro/Code/Drone/oryx/node_modules/cylon-rolling-spider/node_modules/rolling-spider/node_modules/noble/node_modules/xpc-connection
> node-gyp rebuild

  CXX(target) Release/obj.target/binding/src/XpcConnection.o
  SOLINK_MODULE(target) Release/binding.node

> noble@0.3.14 install /Users/alejandro/Code/Drone/oryx/node_modules/cylon-rolling-spider/node_modules/rolling-spider/node_modules/noble
> node-gyp rebuild

  LINK(target) Release/hci-ble
Undefined symbols for architecture x86_64:
  "_main", referenced from:
      start in crt1.10.5.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Release/hci-ble] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/alejandro/Code/Drone/oryx/node_modules/cylon-rolling-spider/node_modules/rolling-spider/node_modules/noble
gyp ERR! node -v v4.1.1
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "cylon-rolling-spider"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! code ELIFECYCLE

npm ERR! noble@0.3.14 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the noble@0.3.14 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the noble package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls noble
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/alejandro/Code/Drone/oryx/npm-debug.log
```

Any ideas what's going on?

**Comment by ChrisTheBaron** [comment]
Hey, 

This looks like it's an issue with the noble package. I'll see if I can replicate the issue and then update to the latest versions to see if it helps. Thanks for bringing it to my attention. 

Cheers, Chris

**Comment by alejandronanez** [comment]
Hey @ChrisTheBaron I just created https://github.com/ChrisTheBaron/cylon-rolling-spider/pull/11 with latest versions of Cylon and Rolling Spider.

**Comment by alejandronanez** [comment]
Fixed on #11
